src/main/kotlin/org/elm/ide/test/run/ElmTestRunProfileState.kt

```diff
@@ -83,14 +89,21 @@ class ElmTestRunProfileState internal constructor(
     }
 
     override fun createConsole(executor: Executor): ConsoleView? {
+        // Should always have an Elm project as we only start a test (causing this method to be called) if we found a project.
+        if (elmProject == null) throw IllegalStateException("Elm project not found")
```

Comment:
No big deal, but I would use Kotlin's `error` function instead of throwing illegal state exception. And I would also get rid of the comment since I think it can be assumed.